pkgs/ort.sh

```diff
@@ -18,6 +18,12 @@
     until git clone -b "$GIT_TAG" "$GIT_REPO"; do echo 'Retrying'; done
     cd onnxruntime
 
+    # Known issues:
+    #  - CUDA test hang without GPU.
```

Comment:
one more space before `-`, i.e. always 1 after `#`; 2 before `-` plus `- ` to make them 4 in total for next lines.

Reply:
sure. but actually this format is from your previous code:

https://github.com/toothache/Roaster/blob/41ef5a01e28667092e9eee345372a8755314e440/pkgs/ort.sh#L21-L23

Reply:
Great! Thanks for bring this to attention.
That one was a typo. Has been fixed for a while.
Also found another one with `git grep` in zstd.sh and fixed: https://github.com/xkszltl/Roaster/commit/e609f562f6049b194070a977a450b6708e1ecb20